src/syntax.ml

```diff
@@ -139,6 +142,10 @@ type p_config =
   (** Sets the configuration for a builtin syntax (e.g., nat literals). *)
   | P_config_binop   of binop
   (** Define (or redefine) a binary operator (e.g., ["+"] or ["×"]). *)
+  | P_config_prover  of string
+  (** Set the prover to use inside a proof *)
+  | P_config_prover_limit of int
```

Comment:
Please add the unit used. It should be in seconds.

Reply:
Done in a799a11d299780b9c79ff5d60c2b6fa8dce2c982

Reply:
Thanks to put "(in seconds)" at the end and add a dot at the end (idem above).

Reply:
Fixed in 5c6ddb2bfc5156d6ffa565334e1d190a4bc49f08.